widget/embedded/src/utils/hub.ts

```diff
@@ -1,22 +1,21 @@
 import type { CommonNamespaceKeys } from '@rango-dev/wallets-core';
-
-import { Namespace } from '@rango-dev/wallets-shared';
+import type { Namespace } from '@rango-dev/wallets-core/namespaces/common';
 
 export function convertCommonNamespacesKeysToLegacyNamespace(
```

Comment:
This function isn't used anywhere. If it isn't needed, it would be better to remove it.

Reply:
good catch! removed the file.

Reply:
I think you forgot to remove it

Reply:
you are right. i pushed a commit right now. https://github.com/rango-exchange/rango-client/pull/969/commits/fda14bd29731b1d67302af2079fbb4c699b00404